iot-e2e-tests/docker/readme.md

```diff
@@ -0,0 +1,35 @@
+# Docker for WSL setup for local test
+
+This script will setup docker images and run instances of these images to support E2E testing.
+
+Before running docker-setup.sh script:
+
+- Install WSL (See installation details here: https://docs.microsoft.com/windows/wsl/install). Reboot after install.
```

Comment:
nit: is it worth specifying which version of WSL to use? I think the default these days is 2 but I'm not entirely sure.

Reply:
Looks like the default is wsl 2, I'm thinking it's not worth specifying since the learn docs we link to don't either except in the specific version section.